android/java/org/mozilla/vrb/etc2/LICENSE

```diff
@@ -0,0 +1,215 @@
+Copyright 2013 Dennis Ippel
```

Comment:
We don't need the whole license, just this:
https://github.com/Rajawali/Rajawali/blob/master/LICENSE.txt